tests/integration/test_pr_agent_workflow_specific.py

```
@@ -0,0 +1,604 @@
+            workflow_raw (str): Raw text content of the GitHub Actions workflow file being validated.
+        """
+        # Capture the action reference after 'uses:' regardless of trailing content
+        uses_pattern = r'^\s*uses:\s*([^\s@]+)@([^\s#]+)'
+        uses_statements = re.findall(uses_pattern, workflow_raw, re.MULTILINE)
+
+        assert uses_statements, "No 'uses:' statements found to validate"
+
+        for owner_repo, ref in uses_statements:
+            # Exclude local or docker actions
+            if not owner_repo or '/' not in owner_repo:
+                def test_checkout_has_fetch_depth(self, workflow_raw: str):
```

Ruff (0.14.7)

Missing return type annotation for private function `test_checkout_has_fetch_depth`

Add return type annotation: `None`

(ANN202)